rav1e_wasm/src/lib.rs

```diff
@@ -0,0 +1,56 @@
+mod utils;
```

Comment:
Please add the license headers to the newly created files

Reply:
Which one?
* rav1e contributors 2017-2020 (e.g. `src/context/block_unit.rs`)
* rav1e contributors 2017-2019 (e.g. `build.rs`)
* AOM 2001-2016 + rav1e contributors 2017-2019 (e.g. `src/capi.rs`)

I guess rav1e contributors 2017-2020?

Reply:
Gotcha